canonical-websites/snapcraft.io
# Remove metrics related code from settings page

Just a branch to cleanup some unnecessary code.

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/<snap_name>/settings
- Make changes to all fields
- Everything should work as expected
